knitware-bots/barr-e/spine/2_idf_build.sh

```
@@ -0,0 +1,7 @@
+clear
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)